topology/University of Hawaii/Information Technology Center/ITC-PRP.yaml

```diff
@@ -0,0 +1,20 @@
+GroupID: 1319
```

Comment:
PRP is no more, long live NRP.

The filename should be NRP, PRP is no longer a thing as of October. 1.

Reply:
Ah good catch; i had only changed the dir name before.  Updated.